### PR TITLE
SET-722 Refactor cci_worker to perform automation smoothly

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/SET-722

### CODE REVIEW

File: cci_worker.yml [hunk 1]
```diff
@@ -52,16 +52,25 @@
...
         - jq
+        - git
       loop_control:
         loop_var: pretaskfiles
 
+  tasks:
+    - ansible.builtin.include_role:
+        name: "{{ taskfiles }}"
+      with_items:
+        - cci_worker
```
Comment (spyrkob) [inline cci_worker.yml:63]: why is this included twice?

File: cci_worker.yml [hunk 2]
```diff
@@ -52,16 +52,25 @@
...
+        loop_var: taskfiles
+
+  post_tasks:
     - name: "Ensure jaegertracing docker image is downloaded for tests"
       containers.podman.podman_image:
         name: "quay.io/jaegertracing/all-in-one:latest"
       become: yes
       become_user: "{{cci_worker_username}}"
 
-  tasks:
```
Comment (spyrkob) [inline cci_worker.yml:64]: do all of those become `post_tasks`? why?

File: roles/cci_worker/tasks/main.yml [hunk 3]
```diff
@@ -100,6 +100,13 @@
         owner: "{{ cci_worker_username }}"
         group: "{{ cci_worker_username }}"
 
+    - name: "Create maven settings directory"
+      ansible.builtin.file:
+        path: "/home/{{ cci_worker_username }}/.m2/"
+        state: directory
+        mode: "0755"
+        owner: "{{ cci_worker_username }}"
```
Comment (spyrkob) [inline roles/cci_worker/tasks/main.yml:108]: add group ownership as well `group: "{{ cci_worker_username }}"`

**Review by spyrkob — CHANGES_REQUESTED** [review]

File: roles/cci_worker/tasks/main.yml (hunk 3 above)
Comment (RanabirChakraborty) [reply roles/cci_worker/tasks/main.yml:108]: Thanks, done!

**Review by spyrkob — APPROVED** [review]
LGTM, thanks
